cluster-up/cluster/ocp-4.3/provider.sh

```diff
@@ -40,7 +40,7 @@ function up() {
     fi
 
     # The auth has the format base64(user:password)
-    auth=$(cat ~/.docker/config.json  | jq -r '.auths["'$container_registry'"]["auth"]' |base64 -d)
+    auth=$(cat ~/.docker/config.json  | docker run --rm -i imega/jq -r '.auths["'$container_registry'"]["auth"]' |base64 -d)
```

Comment:
Can you use a specific tag or a shasum?

Reply:
Add :1.6, I am runing it at check-up-ocp-4.3 job.